forest/cli/src/cli/info_cmd.rs

```diff
@@ -0,0 +1,385 @@
+        let lag = cur_duration_secs - ts;
+
+        let sync_status = if lag < 0 {
+            SyncStatus::Fast
+        } else if lag < EPOCH_DURATION_SECONDS * 3 / 2 {
+            // within 1.5 epochs
+            SyncStatus::Ok
+        } else if lag < EPOCH_DURATION_SECONDS * 5 {
+            // within 5 epochs
+            SyncStatus::Slow
+        } else {
+            SyncStatus::Behind
+        };
+
+        let base_fee = head.min_ticket_block().parent_base_fee().clone();
+
+        // blocks_per_tipset_last_finality = no of blocks till head / chain finality
+        let health = 100. * blocks_per_tipset_last_finality / BLOCKS_PER_EPOCH as f64;
+
+        Self {
+            lag,
+            health,
+            epoch: head.epoch(),
+            base_fee,
+            sync_status,
+            start_time,
+            network,
+            default_wallet_address,
+            default_wallet_address_balance,
+        }
+    }
+
+    fn format(&self, now: DateTime<Utc>) -> Vec<String> {
+        let mut output = Vec::new();
+        let network = format!("Network: {}", self.network);
+
+        let uptime = {
+            let uptime = (now - self.start_time)
+                .to_std()
+                .expect("failed converting to std duration");
+            let uptime = Duration::from_secs(uptime.as_secs());
+            let fmt_uptime = format_duration(uptime);
+            format!(
+                "Uptime: {fmt_uptime} (Started at: {})",
+                self.start_time.with_timezone(&chrono::offset::Local)
+            )
+        };
+
+        let chain = {
+            let base_fee_fmt = self.base_fee.pretty();
+            let lag_time = humantime::format_duration(Duration::from_secs(self.lag.unsigned_abs()));
+            let behind = if self.lag < 0 {
+                format!("{} ahead", lag_time)
+            } else {
+                format!("{} behind", lag_time)
+            };
+
+            format!(
+                "Chain: [sync: {}! ({})] [basefee: {base_fee_fmt}] [epoch: {}]",
+                self.sync_status, behind, self.epoch
+            )
+        };
+
+        let chain_health = format!("Chain health: {:.2}%\n\n", self.health);
+
+        let wallet_info = {
+            let wallet_address = self
+                .default_wallet_address
+                .clone()
+                .unwrap_or("address not set".to_string());
+
+            let wallet_balance = {
+                match balance(&self.default_wallet_address_balance) {
+                    Ok(bal) => format!("[balance: {}]", bal),
+                    Err(e) => e.to_string(),
+                }
+            };
+
+            format!(
+                "Default wallet address: {} [{}]",
+                wallet_address, wallet_balance
+            )
+        };
+
+        output.push(network);
+        output.push(uptime);
+        output.push(chain);
+        output.push(chain_health);
+        output.push(wallet_info);
+
+        output
```

Comment:
Just construct a vec with `vec![..]` here.
It's surprising to return `Vec<String>` - could you add a comment explaining why?

Reply:
was needed for color strings fmt, but not needed atm. Changed it to a `String`